### ISSUE TITLE
Show if projects are NFE

### ISSUE THREAD

**Issue by Secret-chest** [ScratchAddons/ScratchAddons#6484]
### Why this would be helpful

I want to see whether projects are FE, NFE or not rated.

### How this addon works

It just adds a text next to the share date showing the status of the project.

### Additional context

ScratchTools has it.

**Comment by WorldLanguages** [comment]
How exactly do you determine whether a project is NFE?

**Comment by Samq64** [comment]
https://scratch.mit.edu/discuss/topic/150324/

**Comment by WorldLanguages** [comment]
Looks like this is the method that still works: https://scratch.mit.edu/discuss/post/1375754/

**Comment by CST1229** [comment]
> It requires the project to have at least 1 (public? not sure) remix

You can get the status of a project without any remixes by HTML-scraping the normal remixtree page.

**Comment by apple502j** [comment]
I doubt this feature would be useful. Rejected.

**Comment by WorldLanguages** [comment]
Quite a few people would be interested on this feature, I had a Scratch project that explained how to know if a project is NFE through the developer console.
I guess the only reason to actually reject this idea is trying to avoid indirectly causing drama, as it's very uncommon for an NFE decision to be undone. But that doesn't mean people wouldn't find this useful.

**Comment by DNin01** [comment]
Can I hear some examples of how this might come in handy? Also, keep in mind that the proposed implementation is technically an exploit.

**Comment by DNin01** [comment]
It also feels a little invasive to be able to see this information on projects not made by you.

But then why intentionally limit it to only projects you created when Scratch isn't stopping us from using this on any project with at least one remix?

**Comment by WorldLanguages** [comment]
> Also, keep in mind that the proposed implementation is technically an exploit.

Nah, I don't think so. We just need to be careful and not advertise this addon too much. People trying to find this addon should be able to find it, but we ideally shouldn't educate people on what NFE is if they didn't previously know.

**Comment by DNin01** [comment]
Za-Chary, Scratch Team member at the time, made this point [here](https://scratch.mit.edu/discuss/post/4727633/):

> I can confirm that if someone makes a website that allows you to easily see whether or not a project is marked NFE, then that website is not allowed to be linked on the Scratch website.

…and this:

> Nope, we do not notify someone if we mark their project NFE.  The FE/NFE tags were intended to be used silently without public knowledge, but then some Scratchers came by and made our lives a little more difficult for us.

I mean, Scratch Addons is already a browser extension and cannot be mentioned in Scratch, so this won’t really affect our audience/reach, but just consider that this would be something that makes the Scratch Team unhappy. He has a valid concern here. Do we really want to do that in a somewhat UX-first extension used by 100,000+ users? While I won’t decide for you, my opinion would be to _at most_ only allow seeing the status on the user’s own projects, to minimize any resulting drama.

So just keep all that in mind. We’ve rejected various suggestions with complications in the past. I hope what I’ve said makes sense.

Other than that, I do like that the PR tries to use alternative terms instead of “NFE” and “FE”.

**Comment by WorldLanguages** [comment]
Note that there was relevant discussion in PR #6513

**Comment by WorldLanguages** [comment]
> > but then some Scratchers came by and made our lives a little more difficult for us.

Also, just wanted to point out I think this is a bit misleading :P
As someone else said in the forum page you linked:

> Something on the wiki had to recently be removed because of it.

I'm somewhat involved, but it doesn't feel right to say a group of children complicated things by reposting information the Scratch Team themselves made public for the purpose of documenting in the Scratch Wiki.

**Comment by WorldLanguages** [comment]
I also agree about showing whether someone else's project is NFE or not making little sense.
A compromise could be supporting other users' projects but having to check the remix tree / remix list for those.

**Comment by BroJac5246** [comment]
I think it _might_ be okay to show the status of your own project. But Za-Chary made it clear that ST probably won't be happy if this functionality were to be added for any project.

If someone wanted that then it would probably be best in some completely separate, unaffiliated browser extension.

**Comment by BOplaid** [comment]
im switching to scratch tools i dont care at all abt scratchaddons by now
